### DISCUSSION TITLE
Modeling Flame Extinction and Reignition in FDS

### DISCUSSION THREAD

**Discussion by AeonB** [firemodels/fds#13262] [General]
Combustion within a steel rectangular compartment was simulated in FDS. The compartment has one air inlet for forced air and one outlet for exhaust. Fuel is introduced into the compartment with a spray nozzle and is ignited by hot spheres that simulates an ignitor. Dodecane is the fuel that is used for the simulation and AUTO_IGNITION_TEMPERATURE is set to 205 C. My concern is the ability to achieve re-ignition after combustion has subsided due to limited oxygen in the compartment. The air inlet is constantly providing air into the compartment so I would expect the oxygen level get back to a point where combustion is supported? The results from the simulation shows the development of the combustion event, the extinction due to limited oxygen, however no reignition was observed.

**Comment by mcgratta** [comment]
Two issues here. First, try this experiment for real, and see if you can reliably re-ignite hot, fuel-rich, product-laden gases in a compartment. It is difficult to do reliably. Second, examine the model and its assumptions. How about grid resolution? What is the average temperature of a grid cell vs a typical flame or "spark" temperature? 

You are in research territory here, and you have to look at both the physical and numerical aspects of the scenario.

**Reply by AeonB** [reply]
The physical experiment was conducted using Aviation fuel (Jet A) and reignition was achieved. I will review the model assumptions and grid resolution. The aim is to see if FDS can reproduce the phenomena of extinction and reignition. Physical experiment temperatures were above 1000F. FDS produced similar results but no resurgence of combustion was observed in the simulation as it was observed in the physical experiment

**Reply by mcgratta** [reply]
Was there a spark ignitor or hot surface?

**Reply by AeonB** [reply]
A spark ignitor was used as the ignition source and was constantly on throughout the entirety of the experiment. This was done to ensure most combustibles were burned prior to repeat of the next iteration of testing.

**Reply by mcgratta** [reply]
And how are you modeling the spark ignitor? Do you have grid resolution on the order of a mm or less that will resolve the 1800 C, say, temperature of the electrical arc or whatever that spark is? If not, then you have to model the effect of the spark within a grid cell somehow, and that is something that no one, to my knowledge, has figured out how to do.

**Reply by AeonB** [reply]
Unfortunately I am not that experienced with modeling and have not found a way to model the effect of an electrical spark within a grid cell. I differed to simulating four (arbitrary number) hot spheres diameter 0.001m located where the ignitor is positioned in the physical experiment. Combustion was achieved with this method however reignition was not.

**Comment by rmcdermo** [comment]
If you search on "spark" in the user guide, you will find this example:
```
&REAC FUEL='METHANE', AUTO_IGNITION_TEMPERATURE=540 /
&PROP ID='spark', SPARK=T /
&DEVC ID='spark-1', XYZ=..., QUANTITY='TIME', SETPOINT=10, PROP_ID='spark' /
```
This idea here is that within a specified cell (identified by the location of the DEVC) you remove the AIT threshold and allow the fuel and air to burn.  This is more robust than trying to resolve kinetics on any practical LES-like grid resolution.

**Reply by AeonB** [reply]
I saw this example but I was not sure if I understood if it meant that it would simulate a constant spark at the time.

**Reply by rmcdermo** [reply]
You may have to play around with custom control functions to turn the spark on and off.  There is an example for turning a fan on and off in 18.5.5 Control Function: CUSTOM.  Play around with it and come back with questions (though it may take Jason to answer them).

**Reply by AeonB** [reply]
Thank you. I most definitely will give it a shot
